### PR TITLE
Bundle dependencies for a new Terminal module

### PR DESCRIPTION
**Pull request by iliajie**

### CODE REVIEW

File: xterm/index.cgi [hunk 1]
```diff
@@ -181,6 +181,18 @@ my $user = $access{'user'};
 if ($user eq "*") {
 	$user = $remote_user;
 	}
+
+# Switch to given user
```
Comment (jcameron) [inline xterm/index.cgi:185]: How is this change related to bundling dependencies? Please separate it out ..

Comment (iliajie) [reply xterm/index.cgi:185]: Will do.

**Comment by jcameron** [comment]
Can you put the change to switch from SHA1 to SHA into a separate PR? It's a good change, but I like to track commits separately..

**Comment by iliajie** [comment]
> Can you put the change to switch from SHA1 to SHA into a separate PR? It's a good change, but I like to track commits separately..

Done.

**Comment by jcameron** [comment]
I still see changes to makedebian.pl and makerpm.pl in this PR though?

**Comment by iliajie** [comment]
> I still see changes to makedebian.pl and makerpm.pl in this PR though?

Yeah, it's related.

**Comment by jcameron** [comment]
Aren't they related to the switch from SHA1 to SHA, rather than bundling the websockets libraries?

**Comment by iliajie** [comment]
Okay, agreed. Fixed. https://github.com/webmin/webmin/pull/1769/commits/b0142918824c7d855874fb407989058030a23d24

**Comment by jcameron** [comment]
Thanks for the update!
